cmd/podman/tree.go

```diff
@@ -1,23 +1,13 @@
 package main
 
 import (
-	"context"
-	"fmt"
+	"github.com/containers/libpod/pkg/adapter"
 
 	"github.com/containers/libpod/cmd/podman/cliconfig"
-	"github.com/containers/libpod/cmd/podman/libpodruntime"
-	"github.com/containers/libpod/libpod/image"
-	units "github.com/docker/go-units"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
-const (
-	middleItem   = "├── "
-	continueItem = "│   "
-	lastItem     = "└── "
-)
-
```

Comment:
I think all this formatting should stay in the presentation level not be pushed into the library.

You can retrieve the same data structures that will be pushed over varlink and then format and present them. 

Unless some other command needs this formatting?

Reply:
Agree, this should never be used by other commands

Reply:
where exactly do you want it so both local and remote clients can use it? and adapter is not considered "the library" remember?

Reply:
Code can still live in cmd/podman - the output logic should be exactly the same, once we get the right structs transferred